guide/README.md

```diff
@@ -8,6 +8,10 @@ For now, TDS is intended for use by TELUS employees and approved partners.
 
 To view recent and past announcements, see our [Announcements](announcements.html) page.
 
+## TDS V2 to V3 migration
+
+TDS V3 is due to arrive on **September 1st, 2019**! Check out our [migration guide](v3-upgrade.md) for information on how to prepare your application for the upgrade.
```

Comment:
It's coming July 1st, not September 1st, 2019

Can we also indicate what TDS v2 and TDS V3 is (w.r.t the mono package):
v2 = Split Components
v3 = Using Styled Components